diff --git a/node_modules/@react-navigation/routers/lib/module/DrawerRouter.js b/node_modules/@react-navigation/routers/lib/module/DrawerRouter.js
index 989daa6..1348ccb 100644
--- a/node_modules/@react-navigation/routers/lib/module/DrawerRouter.js
+++ b/node_modules/@react-navigation/routers/lib/module/DrawerRouter.js
@@ -1,4 +1,5 @@
 import { nanoid } from 'nanoid/non-secure';
+import { Platform } from 'react-native';
 import TabRouter, { TabActions } from './TabRouter';
 export const DrawerActions = {
   ...TabActions,
@@ -128,7 +129,7 @@ export default function DrawerRouter(_ref) {
         case 'NAVIGATE':
           {
             const result = router.getStateForAction(state, action, options);
-            if (result != null && result.index !== state.index) {
+            if (Platform.OS !== 'web' && result != null && result.index !== state.index) {
               return closeDrawer(result);
             }
             return result;
